reference/src/layout/unions.md

```diff
@@ -138,9 +151,25 @@ assert_eq!(size_of::<U>(), 2);
 # }
 ```
 
-> **Note**: Fields are overlapped instead of laid out sequentially, so 
-> unlike structs there is no "between the fields" that could be filled 
-> with padding.
+#### Interior Padding Footgun
+
+**Major footgun:** On some platform ABIs, such as the obscure ARM64, C unions may also have [interior padding] *within* fields, where a byte is padding in every variant:
```

Comment:
Is "obscure" meant to be sarcastic? I'm honestly not sure so please reword. ;) This is technical writing, after all.

Reply:
Yes, it is. ;) The joke is unpopular apparently, will remove.